src/menu.c

```diff
@@ -122,9 +127,16 @@ void open_browser(const char *in)
                 if (browser_pid2) {
                         exit(0);
                 } else {
-                        char *browser_cmd = g_strconcat(settings.browser, " ", url, NULL);
-                        char **cmd = g_strsplit(browser_cmd, " ", 0);
-                        execvp(cmd[0], cmd);
+                        int argc = 2+g_strv_length(settings.browser_cmd);
+                        char **argv = g_malloc_n(sizeof(char*), argc);
```

Comment:
The documentation of `g_malloc_n` has these 2 reversed, probably doesn't matter anyway but it's good to have it in the correct order.
```
gpointer g_malloc_n (gsize n_blocks, gsize n_block_bytes);
```